astropy/io/fits/hdu/compressed/src/compression.c

```diff
@@ -8,7 +8,7 @@
 #include <quantize.h>
 #include <unquantize.h>
 #include <ricecomp.h>
-
+#include <stdlib.h>
```

Comment:
can you add an inline comment for why this is needed ?

Reply:
Yes, this is to avoid warnings about calloc